packages/render-fidelity-tools/src/workflows/test-fidelity.ts

```diff
@@ -70,9 +70,9 @@ screenshotCreator.captureAndAnalyzeScreenshots(scenarioWhitelist)
       // the test run on.
       const testConfigPath = join(outputDirectory, 'config.json');
       const testConfig = require(testConfigPath);
-      const scenarioCount = testConfig.scenarios.length;
       const failCount = modelViewerFidelityErrors.length;
-      const passCount = scenarioCount - failCount;
+      const passCount = testConfig.scenarios.length;
+      const scenarioCount = failCount + passCount;
```

Comment:
Some errors , like hang there when rendering , didn't find goldens for some render, doesn't have a analysisResult and won't be pushed to the testConfig file, so my former way of counting pass and fail is wrong. So i just change to all passed scenarios goest to testConfig, all senarios with errors goes to modelViewerFidelityErrors.  The shortcoming is when deploying, you cant see the failed sceanrios in the dev page because they are not in the conifg file. 

The other option is to calculate the count using the total scenarios count. The shortcoming for this is when running locally and running the test with specific scenarios, the count is not correct

Tell me which method you like better and im happy to change it.

Reply:
I think I like it the way you've done it here. If any fail, nothing will make it to the dev page anyway since the test will error out. That's good for something facing the public, which this will be shortly.